src/pages/index.js

```diff
@@ -20,327 +24,243 @@ import UserInfo from "../components/UserInfo.js";
+}
 
+//***FUNCTION HANDLING PROFILE EDIT SUBMIT***
+function handleProfileFormSubmit(values) {
+  profileEditPopup.renderSaving(true);
+  api
+    .editProfile(values)
+    .then((res) => {
+      userInfo.setUserInfo({
+        name: res.name,
+        description: res.about,
+      });
+      profileEditPopup.close();
+    })
+    .catch(console.error)
+    .finally(() => {
+      profileEditPopup.renderSaving(false);
+    });
+}
 //***POPUPWITHFORMIMAGE***
-const addCardForm = new PopupWithForm(
+const addCardPopup = new PopupWithForm(
   "#profile-add-modal",
   handleAddImageFormSubmit,
   config
 );
 
-//***PREVIEW IMAGE***
-const modalImagePreview = new PopupWithImage("#image-preview-modal");
+//***FUNCTION ADD IMAGE SUBMIT***
+function handleAddImageFormSubmit(values) {
+  addCardPopup.renderSaving(true);
+  api
+    .addCard(values)
+    .then((res) => {
+      const newCard = renderCard(res);
+      cardsContainer.addItem(newCard);
+      formValidators.addCardForm.resetForm();
+      formValidators.addCardForm.disableSubmit();
+      addCardPopup.close();
+    })
+    .catch(console.error)
+    .finally(() => {
+      addCardPopup.renderSaving(false);
+    });
+}
 
-//***PROFILE EDIT SUBMIT***
-function handleProfileFormSubmit(values) {
-  userInfo.setUserInfo(values);
-  profileEditModal.close();
+//***FORM VALIDATOR CREATOR***
+formList.forEach((form) => {
+  const validator = new FormValidator(form, config);
+  const formName = form.getAttribute("name");
+  validator.enableValidation();
+  formValidators[formName] = validator;
+});
+
+//***RENDER CARD FUNCTION***
+function renderCard(cardData) {
+  // console.log(cardData);
+  const card = new Card(
+    cardData,
+    "#card",
+    handleImageClick,
+    handleDeleteClick,
+    handleLikeClick
+  );
+  return card.generateCard();
 }
 
-//***ADD IMAGE SUBMIT***
+/*/***NEW SECTION***
+const cardsContainer = new Section(
+  { items: initialCards, renderer: renderCard },
+  ".cards__list"
+);*/
+
+//***RENDER CARDSCONTAINER***
+//cardsContainer.rendererItems();
+
+/* **ADD IMAGE SUBMIT***
 function handleAddImageFormSubmit(values) {
   const newCard = renderCard(values);
   cardsContainer.addItem(newCard);
   formValidators.addCardForm.resetForm();
   formValidators.addCardForm.disableSubmit();
   addCardForm.close();
-}
+}*/
 
-//***POPULATE PREVIEW MODAL***
+//***POPULATE PREVIEW POPUP***
 function handleImageClick(name, link) {
-  modalImagePreview.open({ name, link });
+  previewPopup.open({ name, link });
 }
 
-//*** FILL INPUTS OF PROFILE EDIT MODAL ***
+/*/*** FILL INPUTS OF PROFILE EDIT POPUP ***
 function fillProfileInputs() {
   const userCurrentInfo = userInfo.getUserInfo();
   profileInputList[0].value = userCurrentInfo.title;
   profileInputList[1].value = userCurrentInfo.description;
-}
+}*/
 
 //***  CLICK EVENT LISTENER FOR THE EDIT BUTTON***
 profileEditButton.addEventListener("click", () => {
-  fillProfileInputs();
+  const userData = userInfo.getUserInfo();
```

Comment:
* You need to **insert** the user’s data into the inputs when you open the **profile popup** to let the user **edit** the data and submit it.